core/utils/poly-look/src/visualisations/charts/bubble-charts/bubbleCluster/bubbleCluster.js

```diff
@@ -49,17 +53,19 @@ export class BubbleCluster extends Chart {
     width,
     height,
     bubbleColor = defaultBubbleColor,
+    strokeColor = defaultStrokeColor,
     textColor = defaultTextColor,
     opacity = defaultOpacity,
-    showValues = defaultValueShowing,
+    text = defaultText,
     onBubbleClick = defaultOnClickFunction,
     filter,
   }) {
     super({ selector, data, width, height });
     this._bubbleColor = bubbleColor;
+    this._strokeColor = strokeColor;
     this._textColor = textColor;
     this._opacity = opacity;
-    this._showValues = showValues;
+    this._text = text;
```

Comment:
But then it's assigned to this, which is simply text. So what will happen in the default case? Does it ever get the default?